validate_sample.py

```diff
@@ -0,0 +1,367 @@
+import argparse
+import csv
+
+ARID = set()
+
+
+VALID_ESTABLISHMENT_TYPES = {
+    'Household',
+    'Sheltered Accommodation',
+    'Hall of Residence',
+    'Care Home',
+    'Boarding School',
+    'Hotel',
+    'Hostel',
+    'Residential Caravanner',
+    'Gypsy Roma Traveller',
+    'Residential Boater'}
+
+
+VALID_TREATMENT_CODES = {
+    'HH_LF2R1E', 'HH_LF2R2E', 'HH_LF2R3AE', 'HH_LF2R3BE', 'HH_LF3R1E', 'HH_LF3R2E', 'HH_LF3R3AE', 'HH_LF3R3BE',
+    'HH_LFNR1E', 'HH_LFNR2E', 'HH_LFNR3AE', 'HH_LFNR3BE', 'HH_LF2R1W', 'HH_LF2R2W', 'HH_LF2R3AW', 'HH_LF2R3BW',
+    'HH_LF3R1W', 'HH_LF3R2W', 'HH_LF3R3AW', 'HH_LF3R3BW', 'HH_LFNR1W', 'HH_LFNR2W', 'HH_LFNR3AW', 'HH_LFNR3BW',
+    'HH_1LSFN', 'HH_2LEFN', 'HH_QF2R1E', 'HH_QF2R2E', 'HH_QF2R3AE', 'HH_QF3R1E', 'HH_QF3R2E', 'HH_QF3R3AE',
+    'HH_QFNR1E', 'HH_QFNR2E', 'HH_QFNR3AE', 'HH_QF2R1W', 'HH_QF2R2W', 'HH_QF2R3AW', 'HH_QF3R1W', 'HH_QF3R2W',
+    'HH_QF3R3AW', 'HH_QFNR1W', 'HH_QFNR2W', 'HH_QFNR3AW', 'HH_3QSFN'}
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Load a sample file into response management.')
+    parser.add_argument('sample_file_path', help='path to the sample file', type=str)
+    parser.add_argument("-v", "--verbose", help="Turn on all warnings", action="store_true")
+    return parser.parse_args()
+
+
+args = parse_arguments()
+
+
+def validate_header_row(sample_file_path):
+    with open(sample_file_path) as f:
+        first_line = f.readline()
+        result = [x.strip() for x in first_line.split(',')]
```

Comment:
You can get the header from the `DictReader` with the `fieldnames` property.

Reply:
I've changed that.